skipper.go

```diff
@@ -955,6 +1004,16 @@ func listenAndServe(proxy http.Handler, o *Options) error {
 	return listenAndServeQuit(proxy, o, nil, nil, nil)
 }
 
+func initGrant(c *auth.OAuthConfig, o *Options) error {
+	if err := c.Init(); err != nil {
+		log.Errorf("Error while initializing oauth grant filters: %v.", err)
```

Comment:
Error while initializing .. -> Failed to initialize ..

Reply:
Fixed.